src/test/java/org/broadinstitute/hellbender/tools/walkers/variantutils/SelectVariantsIntegrationTest.java

```diff
@@ -0,0 +1,751 @@
+    public void testKeepOriginalACAndENV() throws IOException {
+        final String testFile = getToolTestDataDir() + "vcfexample.loseAlleleInSelection.vcf";
+
+        final IntegrationTestSpec spec = new IntegrationTestSpec(
+                baseTestString(" --keepOriginalAC -sn NA12892 -env -trimAlternates ", testFile),
+                Collections.singletonList(getToolTestDataDir() + "expected/" + "testSelectVariants_KeepOriginalACAndENV.vcf")
+        );
+
+        spec.executeTest("testKeepOriginalACAndENV--" + testFile, this);
+    }
+
+    @Test(groups={"requiresRemoteAccess"}, enabled=remoteTestsEnabled)
+    public void testKeepOriginalDP() throws IOException {
+        final String testFile = getToolTestDataDir() + "CEUtrioTest.vcf";
+
+        final IntegrationTestSpec spec = new IntegrationTestSpec(
+                        "-R " + b37KGReference
+                            + " --variant " + testFile
+                            + " -o %s "
+                            + " --keepOriginalDP -sn NA12892 ",
+                Collections.singletonList(getToolTestDataDir() + "expected/" + "testSelectVariants_KeepOriginalDP.vcf")
+        );
+
+        spec.executeTest("testKeepOriginalDP--" + testFile, this);
+    }
+
+    @Test(groups={"requiresRemoteAccess"}, enabled=remoteTestsEnabled)
+    public void testMultipleRecordsAtOnePosition() throws IOException {
+        final String testFile = getToolTestDataDir() + "selectVariants.onePosition.vcf";
+
+        final IntegrationTestSpec spec = new IntegrationTestSpec(
+                baseTestString(" -select 'KG_FREQ < 0.5' ", testFile),
+                Collections.singletonList(getToolTestDataDir() + "expected/" + "testSelectVariants_MultipleRecordsAtOnePosition.vcf")
+        );
+
+        spec.executeTest("testMultipleRecordsAtOnePosition--" + testFile, this);
+    }
+
+    @Test(groups={"requiresRemoteAccess"}, enabled=remoteTestsEnabled)
+    public void testNoGTs() throws IOException {
+        final String testFile = getToolTestDataDir() + "vcf4.1.example.vcf";
+
+        final IntegrationTestSpec spec = new IntegrationTestSpec (
+                " -R " + b37KGReference
+                    + " --variant " + testFile
+                    + " -o %s ",
+                Collections.singletonList(getToolTestDataDir() + "expected/" + "testSelectVariants_NoGTs.vcf")
+        );
+
+        spec.executeTest("testNoGTs--" + testFile, this);
+    }
+
+    // @TODO: This test passed using the GATK test files before sequence dictionary validation was enabled in HB
+    // but now fails with: UserException$LexicographicallySortedSequenceDictionary: A USER ERROR has occurred:
+    // Lexicographically sorted human genome sequence detected in variants. From debugging it appears that the
+    // variants file the source of the error message, but it still fails the same way even after running the
+    // picard SortVcf tool on the .vcf using the b37KGReference reference dictionary.
```

Comment:
Can you edit the sequence dictionary manually? You just need to ensure that chromosome 10 comes after chromosomes 1 and 2.